src/js/fluxcontroller.js

```diff
@@ -34,14 +34,53 @@ define(function (require, exports, module) {
 
     var locks = require("./locks"),
         events = require("./events"),
-        storeIndex = require("./stores/index"),
-        actionIndex = require("./actions/index"),
         AsyncDependencyQueue = require("./util/async-dependency-queue"),
         synchronization = require("./util/synchronization"),
         performance = require("./util/performance"),
         log = require("./util/log");
 
     /**
+     * Automatically loads all js files in ./actions folder
+     * replacing their paths with dot separated keys
+     *
+     * @param {function} requireContext require.context call of Webpack pointing to the actions folder
+     *
+     * @return {function()} when called, returns all action modules loaded
+     */
+    var requireActions = function (requireContext) {
+        return requireContext.keys().reduce(function (actionMap, actionKey) {
+            var actionId = actionKey.substring(2, actionKey.indexOf(".js")).replace(/\//gi, ".");
```

Comment:
Can you add example values of the `actionKey` and `actionId` as comment? I think it is easier to understand the purpose of the string transformation with an example.